js/helpers.mjs

```diff
@@ -1,43 +1,63 @@
-
 const pageContainerEl = document.querySelector('.page');
 const textareaEl = document.querySelector('.page > .textarea');
 const overlayEl = document.querySelector('.page > .overlay');
 
+export const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
 
-const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
-
-function applyPaperStyles() {
+export const applyPaperStyles = () => {
   pageContainerEl.style.border = 'none';
   pageContainerEl.style.background = 'linear-gradient(to right,#eee, #ddd)';
-  overlayEl.style.background = `linear-gradient(${Math.random()*360}deg, #0008, #0000)`
+  overlayEl.style.background = `linear-gradient(${
+    Math.random() * 360
+  }deg, #0008, #0000)`;
   overlayEl.style.display = 'block';
   textareaEl.classList.add('paper');
-}
+};
 
-function removePaperStyles() {
+export const removePaperStyles = () => {
   pageContainerEl.style.border = '1px solid #ccc';
   pageContainerEl.style.background = 'linear-gradient(to right,#fff, #fff)';
   overlayEl.style.display = 'none';
   textareaEl.classList.remove('paper');
-}
+};
 
-function addFontFromFile(fileObj) {
+export const addFontFromFile = (fileObj) => {
```

Comment:
This file has a lot of functions so isn't that better to have functions export at the end?

If I want to import a function somewhere, with this code I will have to go through the whole file to see which functions are exported whereas if we have single `export {}` at the end I can just look up and see which functions are exported.

Reply:
Okay makes sense